### ISSUE TITLE
Bug: Page randomly refuses to load

### ISSUE THREAD

**Issue by alexhorn** [mattermost/focalboard#3739]
## Steps to reproduce the behavior

Directly open one of the boards using a bookmark.

## Expected behavior

Usually the page loads, but randomly it just stays blank.

## Edition and Platform

 - Edition: Personal Server
 - Version: v7.2.0 in Docker
 - Browser and OS: Chrome on Windows

## Additional context

I'm running the Personal Server in Docker behind Traefik v2.8.

The browser console shows a WebSocket error. Presumably there's a race condition where it tries to send data before the WebSocket connection is fully connected:

```
main.js?06b36e45a5ccec2d1c93:350 Uncaught (in promise) DOMException: Failed to execute 'send' on 'WebSocket': Still in CONNECTING state.
    at o.sendCommand (https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:350:682844)
    at o.sendUnsubscribeToTeamCommand (https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:350:683221)
    at o.unsubscribeToTeam (https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:350:689024)
    at https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:350:444286
    at Nl (https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:300:103965)
    at t.unstable_runWithPriority (https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:350:3768)
    at Wa (https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:300:44750)
    at jl (https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:300:103563)
    at Fl (https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:300:95091)
    at https://focalboard.example.com/static/main.js?06b36e45a5ccec2d1c93:300:44973
```

**Comment by kamre** [comment]
Looks similar to this bug that was fixed recently:
- #3589

**Comment by wuwinson** [comment]
Fixed with #3718, feel free to re-open if it's still an issue. Thanks!
